src/ResourceManager/Relay/Commands.Relay.Test/ScenarioTests/Common.ps1

```diff
@@ -0,0 +1,72 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+<#
+.SYNOPSIS
+Gets a resource group name for testing.
+#>
+function Get-ResourceGroupName
+{
+    return getAssetName
+}
+
+
+<#
+.SYNOPSIS
+Gets the location for the Website. Default to West US if none found.
+#>
+function Get-Location
+{
+	if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback)
+	{
+		$namespace = "Microsoft.Web"
+		$type = "sites"
+		$location = Get-AzureRmResourceProvider -ProviderNamespace $namespace | where {$_.ResourceTypes[0].ResourceTypeName -eq $type}
+  
+		if ($location -eq $null) 
+		{  
+			return "West US"  
+		} else 
+		{  
+			return $location.Locations[0]  
+		}
+	}
+
+	return "West US"
+}
+
+<#
+.SYNOPSIS
+Gets the location for the Website. Default to West US if none found.
+#>
+function Get-SecondaryLocation
+{
+	if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback)
```

Comment:
Resolved, removed the common.ps1 from test folder and passed Location in script files.